app/src/main/java/com/anytypeio/anytype/ui/home/HomeScreenFragment.kt

```diff
@@ -73,6 +73,51 @@ class HomeScreenFragment : BaseComposeFragment(),
 
     private val vm by viewModels<HomeScreenViewModel> { factory }
 
+    private var spaceSwitchCount: Int = 0
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        val tooltipShownOnce = isTooltipShown()
+        spaceSwitchCount = getSpaceSwitchCount()
+        if (!tooltipShownOnce && spaceSwitchCount >= 2) {
+            vm.showTooltip()
+            saveTooltipShown(true)
+            spaceSwitchCount = 0
+        } else if (!tooltipShownOnce) {
+            spaceSwitchCount++
+        }
+        saveSpaceSwitchCount(spaceSwitchCount)
+    }
+
+    private fun saveTooltipShown(shown: Boolean) {
+        val sharedPreferences =
```

Comment:
consider to use DefaultUserSettingsCache. sharedPreferences should not be presented in Fragment